formulas/wick-base/functions/wick-service-add-dependency-if-enabled

```diff
@@ -11,15 +11,14 @@
 #
 # Returns the result of wickServiceAddDependency.
 wickServiceAddDependencyIfEnabled() {
-    local dependency result service
+    local dependency service
 
-    wickGetArgument service 0 "$@"
-    wickGetArgument dependency 1 "$@"
-    wickStrictRun result wickServiceIsEnabled "$dependency"
+    wickGetArgument service 0 "$@" || return $?
+    wickGetArgument dependency 1 "$@" || return $?
 
-    if [[ "$result" -eq 0 ]]; then
-        wickServiceAddDependency "$service" "$dependency"
+    if wickServiceIsEnabled "$dependency"; then
+        wickServiceAddDependency "$service" "$dependency" || return $?
     else
-        wickInfo "Service $dependency does not exist.  No adding it as a dependency"
+        wickInfo "Service $dependency does not exist.  No adding it as a dependency" || return $?
```

Comment:
I was raised with two spaces after punctuation.  That rule has been changed to one space after punctuation, though I wasn't going through all of the code and removing the double space stuff.  Want me to make this change?